docs/static/performance-checklist.asciidoc

```diff
@@ -49,59 +50,132 @@ sure-fire way to create a confusing situation.
+  Set the JVM heap space accordingly in the `jvm.options` config file (See <<config-setting-files>> for more info).
 
 * Measure each change to make sure it increases, rather than decreases, performance.
 
-* Ensure that you leave enough memory available to cope with a sudden increase in event size. For example, an application that generates exceptions that are represented as large blobs of text.
+* Ensure that you leave enough memory available to cope with a sudden increase in event size.
+  For example, an application that generates exceptions that are represented as large blobs of text.
 
 * The number of workers may be set higher than the number of CPU cores since outputs often spend idle time in I/O wait conditions.
 
 * Threads in Java have names and you can use the `jstack`, `top`, and the VisualVM graphical tools to figure out which resources a given thread uses.
 
-* On Linux platforms, Logstash labels all the threads it can with something descriptive. For example, inputs show up as `[base]<inputname`, and pipeline workers show up as `[base]>workerN`, where N is an integer.  Where possible, other threads are also labeled to help you identify their purpose.
+* On Linux platforms, Logstash labels all the threads it can with something descriptive.
+  For example, inputs show up as `[base]<inputname`, and pipeline workers show up as `[base]>workerN`, where N is an integer.
+  Where possible, other threads are also labeled to help you identify their purpose.
 
 [float]
 [[profiling-the-heap]]
-==== Profiling the Heap
+==== Profiling the heap
 
-When tuning Logstash you may have to adjust the heap size. You can use the https://visualvm.github.io/[VisualVM] tool to profile the heap. The *Monitor* pane in particular is useful for checking whether your heap allocation is sufficient for the current workload. The screenshots below show sample *Monitor* panes. The first pane examines a Logstash instance configured with too many inflight events. The second pane examines a Logstash instance configured with an appropriate amount of inflight events. Note that the specific batch sizes used here are most likely not applicable to your specific workload, as the memory demands of Logstash vary in large part based on the type of messages you are sending.
+When tuning Logstash you may have to adjust the heap size.
+You can use the https://visualvm.github.io/[VisualVM] tool to profile the heap.
+The *Monitor* pane in particular is useful for checking whether your heap allocation is sufficient for the current workload.
+The screenshots below show sample *Monitor* panes.
+The first pane examines a Logstash instance configured with too many inflight events.
+The second pane examines a Logstash instance configured with an appropriate amount of inflight events.
+Note that the specific batch sizes used here are most likely not applicable to your specific workload, as the memory demands of Logstash vary in large part based on the type of messages you are sending.
```

Comment:
not worth changing now, but typically I prefer to put style changes in a separate PR to facilitate reviewing, especially in documentation where github isn't great at showing diffs.